tests/Framework.py

```diff
@@ -298,13 +299,6 @@ def setUp(self):
                 if GithubCredentials.jwt
                 else None
             )
-            self.app_auth = (
-                github.Auth.AppAuth(
-                    GithubCredentials.app_id, GithubCredentials.app_private_key
-                )
-                if GithubCredentials.app_id and GithubCredentials.app_private_key
-                else None
-            )
```

Comment:
No need to remove this existing code.

Reply:
I had added this, do you want me to re-add it even if this is not used here?

Reply:
This diff tells me, this was there before and your pull-request removes it.

Reply:
This is the last bit left. I'd say there is no need to remove this code by this pull request.

Reply:
hum, I was sure I had added that when I was creating a new test class.... I may have it mixed up with something else. Will re-introduce.